packages/api/package.json

```
@@ -67,7 +67,7 @@
 			"name": "API (top-level class)",
 			"path": "./lib-esm/index.js",
 			"import": "{ Amplify, API }",
-			"limit": "89 kB"
+			"limit": "89.79 kB"
```

I think we should just give ourselves a few kb of margin while we work. Pick a value that'll signal us if we do something downright silly. But otherwise we can backfill and set a target value later, IMHO.

```suggestion
			"limit": "95 kB"
```